README.md

```diff
@@ -130,36 +130,6 @@ pipenv shell
 
 You have now created a virtual environment for this project. You can read more about virtual environments [here](https://realpython.com/pipenv-guide/).
```

Comment:
the dunesql readme draft was updated here if we want to use that version https://github.com/duneanalytics/spellbook-trino/pull/44